api/src/core/services/comment/comment.service.ts

```diff
@@ -68,4 +71,61 @@ export class CommentService {
       comment
     };
   };
+
+  public getComments = async (
+    dto: GetCommentsDto
+  ): Promise<GetCommentsResponse> => {
+    const pageSize = dto.pageSize ? dto.pageSize : 10;
```

Comment:
`api/test/core/services/comment/comment.service.test.ts` L35 에서 처럼

```suggestion
    const pageSize = dto.pageSize ?? 10;
```

이렇게 하면 더 간단할 것 같아요

Reply:
좋은 제안 감사합니다. https://github.com/MovieReviewComment/Mr.C/pull/192/commits/53d74b7dd33ea407ac874790d574878ab25ffaf1 에서 반영했습니다.😊